### PR TITLE
Provide npcap distribution internally

### PR DESCRIPTION
**Pull request by v1v**
### What

Provide the npcap distribution in an internal docker image, for such, it uses the internal docker registry, the main docker image and copy the file to `/installer` folder.

### Actions

- [x] Verifies no security disclosure.
- [x] Add `npcap` version in the tag image.

#### Build

image

image

#### Push

image

#### Pull

image

#### Files

image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,20 @@
 TARGETS=go1.16 go1.17
 ARM_TARGETS=go1.16 go1.17
+# TODO: to be replaced once we validate it works as expected
+# TODO: version to be tagged too.
+#NPCAP_FILE=npcap-1.60-oem.exe
+NPCAP_FILE=test.txt
```
Comment (v1v) [inline Makefile:6]: This will be replaced in a follow up, once we validate everything was pushed as expected

File: Makefile.common [hunk 2]
```diff
@@ -3,14 +3,25 @@ VCS_REF       := $(shell git rev-parse HEAD)
 VCS_URL       := https://github.com/elastic/golang-crossbuild
 BUILD_DATE    := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 .DEFAULT_GOAL := build
+SUFFIX_NPCAP_VERSION := -npcap-$(NPCAP_VERSION)
+NPCAP_REPOSITORY := docker.elastic.co/observability-ci
 
-# Requires login at https://docker.elastic.co:7000/.
 push:
-	echo ">> Pushing $(REPOSITORY)/$(NAME):$(VERSION)$(SUFFIX)$(TAG_EXTENSION)"
+	@[ "$(SUFFIX)" = "$(SUFFIX_NPCAP_VERSION)" ] \
```
Comment (v1v) [inline Makefile.common:10]: I could not find the way to use `ifeq` with SUFFIX and SUFFIX_NPCAP_VERSION, even though these values were the same, but the `ifeq` returned false.

I guess, something should be related to where the SUFFIX is defined and the `include` function.

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,23 @@
 TARGETS=go1.16 go1.17
 ARM_TARGETS=go1.16 go1.17
+# TODO: to be replaced once we validate it works as expected
+# TODO: version to be tagged too.
+NPCAP_VERSION=1.60
+#NPCAP_FILE=npcap-$(NPCAP_VERSION)-oem.exe
+NPCAP_FILE=test.txt
```
Comment (v1v) [inline Makefile:7]: TODO: to be replaced once we agree this is the way to go

```suggestion
NPCAP_VERSION=1.60
NPCAP_FILE=npcap-$(NPCAP_VERSION)-oem.exe
```

**Review by mdelapenya — APPROVED** [review]

File: Makefile.common [hunk 4]
```diff
@@ -3,14 +3,29 @@ VCS_REF       := $(shell git rev-parse HEAD)
 VCS_URL       := https://github.com/elastic/golang-crossbuild
 BUILD_DATE    := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 .DEFAULT_GOAL := build
+NPCAP_VERSION := 1.60
+# TODO: to be replaced once we validate it works as expected
+#NPCAP_FILE   := npcap-$(NPCAP_VERSION)-oem.exe
+NPCAP_FILE    := test.txt
```
Comment (v1v) [inline Makefile.common:9]:
```suggestion
NPCAP_FILE   := npcap-$(NPCAP_VERSION)-oem.exe
```

File: NPCAP.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# NPCAP
+
+If you'd like to bump the npcap version please follow the below steps:
+
+1) Update `NPCAP_VERSION` value in the `Makefile`.
```
Comment (v1v) [inline NPCAP.md:5]:
```suggestion
1) Update `NPCAP_VERSION` value in the `Makefile.common`.
```

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**